cloudify/agentfile.go

```diff
@@ -52,8 +49,39 @@ func (cl *Client) updateHostFromAgent() {
 	}
 }
 
-func (cl *Client) printf(format string, v ...interface{}) {
+func (cl *Client) debugLogf(format string, v ...interface{}) {
 	if cl.Debug {
 		log.Printf(format, v...)
 	}
 }
+
+// ValidateBaseConnection - check configuration params (without tenant)
+func ValidateBaseConnection(cloudConfig ClientConfig) error {
+	if len(cloudConfig.Host) == 0 && len(cloudConfig.AgentFile) == 0 {
+		return fmt.Errorf("You have empty host")
+	}
+
+	if len(cloudConfig.User) == 0 {
+		return fmt.Errorf("You have empty user")
+	}
+
+	if len(cloudConfig.Password) == 0 {
+		return fmt.Errorf("You have empty password")
+	}
+
```

Comment:
@0lvin What about checking the agent/tenant values ?

Reply:
We don't need to check "tenant" globally, you can use command like:
````
bin/cfy-go tenants list -host https://<some_host> -tenant "" -debug
````

Reply:
Got it :)